app/src/main/java/sdp/moneyrun/menu/MainLeaderboardListAdapter.java

```diff
@@ -0,0 +1,49 @@
+package sdp.moneyrun.menu;
+
+import android.app.Activity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+
+import sdp.moneyrun.R;
+import sdp.moneyrun.player.Player;
+
+public class MainLeaderboardListAdapter extends ArrayAdapter<Player> {
+
+    private final String[] rank = {"\uD83E\uDD47", "\uD83E\uDD48", "\uD83E\uDD49"};
```

Comment:
these values are not really straight forward a comment about what each value represents would be useful

Reply:
done